src/storage.js

```diff
@@ -1,6 +1,20 @@
 class LocalStorageStore {
   getItem(key) {
-    return JSON.parse(localStorage.getItem(key));
+    let item;
+    try {
+      item = localStorage.getItem(key);
+      return JSON.parse(item);
+    } catch (e) {
+      // old versions of the wallet would save strings without converting
+      // to JSON, so we catch this exception here and return the string directly
+      if (e instanceof SyntaxError) {
+        // first save in JSON format
+        this.setItem(key, item);
```

Comment:
Even though this solution fixes the issues you've seen, it has not been tested in different scenarios and may fail.

For now, I think we must include a `// FIXME` comment saying that it is a temporary solution with a link to an issue to version the localStorage and include migration tests between different versions. Then, we can go forward with this PR.

Reply:
I agree

Reply:
Yes, the idea is not catching every possible error, just the one we know.

https://github.com/HathorNetwork/hathor-wallet-lib/issues/19